spec/controllers/admin/rentals_controller_spec.rb

```diff
@@ -16,15 +16,38 @@
   end
 
   describe 'GET #index' do
-    before do
-      expect(Rental).to receive(:synchronize).with(scope: account) do
-        # pretending to sync rentals :P
-        rental
+    context "with default synchronization" do
+      before do
+        expect(Rental).to receive(:synchronize).with(scope: account) do
+          # pretending to sync rentals :P
+          rental
+        end
```

Comment:
```suggestion
        expect(Rental).to receive(:synchronize).with(scope: account).and_return(rental) # pretending to sync rentals :P
```
you probably don't need a block, also this is a non-blocking comment 🥁

Reply:
it seems to be evaluated before running get :index, so count doesn't change

Reply:
I see, yeah without a block it would be evaluated before.. nevermind then 😅